docs/reference/alert-schema.asciidoc

```diff
@@ -0,0 +1,120 @@
+|{ecs-ref}/ecs-error.html[`error.*`]| `error.*` |ECS `error.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-event.html[`event.*`] | `event.*`| ECS `event.*` fields copied from the source document, if present, for custom query and indicator match rules.
+
+NOTE: categorization fields above (`event.kind`, `event.category`, `event.type`, `event.outcome`) are listed separately above.
+
+|{ecs-ref}/ecs-file.html[`file.*`]| `file.*` | ECS `file.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-group.html[`group.*`] | `group.*` | ECS `group.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-host.html[`host.*`] | `host.*` |ECS `host.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-http.html[`http.*`]| `http.*`| ECS `http.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-log.html[`log.*`]| `log.*` | ECS `log.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-network.html[`network.*`]|`network.*`| ECS `network.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-observer.html[`observer.*`]|`observer.*`|ECS `observer.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-orchestrator.html[`orchestrator.*`] | `orchestrator.*`| ECS `orchestrator.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-organization.html[`organization.*`]| `organization.*`| ECS `organization.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-package.html[`package.*`]|`package.*`|ECS `package.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-process.html[`process.*`]| `process.*`| ECS `process.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-registry.html[`registry.*`] | `registry.*`| ECS `registry.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-related.html[`related.*`] |`related.*` |ECS `related.*` fields copied from the source document, if present, for custom query and indicator match rules.
+|{ecs-ref}/ecs-rule.html[`rule.*`]| `rule.*`| ECS `rule.*` fields copied from the source document, if present, for custom query and indicator match rules.
+
+NOTE: These fields are not related to the detection rule that generated the alert.
```

Comment:
It's not obvious to me whether this references the previous or the following fields, especially since the "Description" column values are so similar before and after the note. Could instead say "The preceding" or "The following" fields.

Reply:
@benironside this is in the same table cell, so it's pertaining to line 51.